packages/teams-js/src/internal/communication.ts

```diff
@@ -587,3 +771,15 @@ function createMessageEvent(func: string, args?: any[]): MessageRequest {
     args: args || [],
   };
 }
+
+/**
```

Comment:
how do you think to put this in a util class?

Reply:
@TrevorJoelHarris  same idea here, would you want me to move it to another file which might be called telemetryUtil.ts?

Reply:
This function is moved to telemetry.ts file as support function for telemetry.